hipe4ml/analysis_utils.py

```diff
@@ -164,7 +163,8 @@ def train_test_generator(data_list, labels_list, sliced_df=False, **kwds):
         del data_list, data_frame
         df_tot_train_test = pd.concat(df_list, sort=True)
         del df_list
-        train_test = train_test_split(df_tot_train_test, np.array(labels_train_test), **kwds)
+        train_test = train_test_split(
+            df_tot_train_test, np.array(labels_train_test), **kwds)
```

Comment:
It looks like your pylint is not picking up the configuration file of the repo where the max line length is 120.

Reply:
It was actually due to the vscode auto-formatter. Fixed now